napari/benchmarks/benchmark_qt_viewer_labels.py

```diff
@@ -40,14 +40,18 @@ def setup(self):
             pos=(500, 500),
             view_direction=None,
         )
+        if NAPARI_0_4_19:
```

Comment:
Do we support anything before this version? If not, this is fine.

Reply:
No, in benchmarks this is the earliest we want to get to (and we can probably start thinking about dropping this too).